src/main/scala/uk/gov/nationalarchives/consignmentexport/Main.scala

```diff
@@ -44,6 +46,7 @@ object Main extends CommandIOApp("tdr-consignment-export", "Exports tdr files in
         _ <- bashCommands.runCommand(s"sha256sum $tarPath > $tarPath.sha256")
         _ <- s3Files.uploadFiles(config.s3.outputBucket, consignmentId, tarPath)
         _ <- graphQlApi.updateExportLocation(config, consignmentId, s"s3://${config.s3.outputBucket}/$consignmentId.tar.gz", exportDatetime)
+        _ <- stepFunction.publishSuccess(taskToken, ExportOutput())
```

Comment:
Don't you need to populate the `ExportOutput` class with the values. It's just sending the defaults as it is I think?

Reply:
Yeah, its just defaults at the moment. Need both your's (https://github.com/nationalarchives/tdr-consignment-export/pull/12/) and Loki's changes in, so I can get the user id and consignment reference from the consignment metadata

Reply:
Ah got you, cheers